setup.py

```diff
@@ -55,6 +55,8 @@
 GROUP_DEFAULT = 'loris'
 GROUP_HELP = 'Group that will own the application and has permission to write to caches. [Default: %s]' % (USER_DEFAULT,)
 
+OVERRIDE_DEPENDENCIES = bool(os.environ.get('OVERRIDE_LORIS_DEPENDENCIES', False))
```

Comment:
The title of the PR doesn't match the variable name, I think the variable name is the right one

Reply:
Yes your right, have updated the PR name.